### PR TITLE
Sfdk: Make package-search alias of package-list

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: share/qtcreator/sfdk/modules/30-building-maintain.json [hunk 1]
```diff
@@ -22,10 +22,10 @@
...
-            "trSynopsis": "[tooling|target] list [-a|--available] [--slow]\n[tooling|target] update <name>\n[tooling|target] register [--user <user> [--password <password>]] {--all|<name>}\n[tooling|target] install <name>\n[tooling|target] install-custom <name> {<URL>|<file>} [--tooling <tooling>] [--no-snapshot]\n[tooling|target] remove [--snapshots-of] <name>\n[tooling|target] package-list <name> [<pattern>...]\n[tooling|target] package-install <name> <package>...\n[tooling|target] package-remove <name> <package>...\n[tooling|target] exec <name> [<command> [<arg>...]]",
+            "trSynopsis": "[tooling|target] list [-a|--available] [--slow]\n[tooling|target] update <name>\n[tooling|target] register [--user <user> [--password <password>]] {--all|<name>}\n[tooling|target] install <name>\n[tooling|target] install-custom <name> {<URL>|<file>} [--tooling <tooling>] [--no-snapshot]\n[tooling|target] remove [--snapshots-of] <name>\n[tooling|target] package-list <name>\n[tooling|target] package-search <name> [<pattern>...]\n[tooling|target] package-install <name> <package>...\n[tooling|target] package-remove <name> <package>...\n[tooling|target] exec <name> [<command> [<arg>...]]",
```
Comment (martyone) [inline share/qtcreator/sfdk/modules/30-building-maintain.json:25]: I would prefer to have just one option, so I suggest to deprecate the original one and drop it from docs immediately.

File: share/qtcreator/sfdk/modules/30-building-maintain.json [hunk 2]
```diff
@@ -22,10 +22,10 @@
...
-            "trSynopsis": "[tooling|target] list [-a|--available] [--slow]\n[tooling|target] update <name>\n[tooling|target] register [--user <user> [--password <password>]] {--all|<name>}\n[tooling|target] install <name>\n[tooling|target] install-custom <name> {<URL>|<file>} [--tooling <tooling>] [--no-snapshot]\n[tooling|target] remove [--snapshots-of] <name>\n[tooling|target] package-list <name> [<pattern>...]\n[tooling|target] package-install <name> <package>...\n[tooling|target] package-remove <name> <package>...\n[tooling|target] exec <name> [<command> [<arg>...]]",
+            "trSynopsis": "[tooling|target] list [-a|--available] [--slow]\n[tooling|target] update <name>\n[tooling|target] register [--user <user> [--password <password>]] {--all|<name>}\n[tooling|target] install <name>\n[tooling|target] install-custom <name> {<URL>|<file>} [--tooling <tooling>] [--no-snapshot]\n[tooling|target] remove [--snapshots-of] <name>\n[tooling|target] package-list\n[tooling|target] package-search <name> [<pattern>...]\n[tooling|target] package-install <name> <package>...\n[tooling|target] package-remove <name> <package>...\n[tooling|target] exec <name> [<command> [<arg>...]]",
```
Comment (martyone) [inline share/qtcreator/sfdk/modules/30-building-maintain.json:25]: I would prefer omitting the old option from the docs as we did on sdk-assistant side.

Comment (vigejolla) [reply share/qtcreator/sfdk/modules/30-building-maintain.json:25]: I thought I already did that, but clearly I didn't..

Comment (vigejolla) [reply share/qtcreator/sfdk/modules/30-building-maintain.json]: maybe now :)

Comment (martyone) [reply share/qtcreator/sfdk/modules/30-building-maintain.json]: One more occurrence in the long help :)

Comment (vigejolla) [reply share/qtcreator/sfdk/modules/30-building-maintain.json]: It's only in the sentence "The 'package-list' command is deprecated, use 'package-search' instead.", which I think is still good to have.

Comment (martyone) [reply share/qtcreator/sfdk/modules/30-building-maintain.json]: We have the warning that is issued when the deprecated option is used.

Comment (vigejolla) [reply share/qtcreator/sfdk/modules/30-building-maintain.json]: Indeed we do. I had already forgotten that.

Comment (vigejolla) [reply share/qtcreator/sfdk/modules/30-building-maintain.json]: another try :)

**Review by martyone — APPROVED** [review]
LGTM, I will appreciate if you wait a bit with merging, so that I do not need to rebase all my branches :)

**Pull request merged**